src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -175,7 +176,7 @@ private static async Task<JObject> GetFunctionConfig(FunctionMetadata metadata,
 
         private static async Task<JObject> GetFunctionConfigFromFile(string path)
         {
-            return JObject.Parse(await FileUtility.ReadAsync(path));
+            return JObject.Parse(Sanitizer.Sanitize(await FileUtility.ReadAsync(path)));
```

Comment:
synctrigger endpoint and /functions/{functionname} endpoint code reads the file if it is present. (Valid for custom handlers /non worker indexing use case).